src/View/Helper/PaginatorHelper.php

```diff
@@ -1169,4 +1170,37 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param int|null $default Default limit for option selecting. Default value is $this->param('perPage').
+     * @param array $options Options for Select tag attributes like class, id or event
+     *
+     * @return string html output.
+     */
+    public function limitControl(array $limits = [], $default = null, array $options = [])
+    {
+        $out = $this->Form->create(null, ['type' => 'get']);
+
+        if (empty($default) || !is_numeric($default)) {
+            $default = $this->param('perPage');
+        }
+
+        if (empty($limits)) {
+            $limits += ['0' => __('All'), '20' => '20', '50' => '50', '100' => '100'];
```

Comment:
That's probably because you haven't added any tests case calling the method without any params.